packages/nocms/package.json

```diff
@@ -38,7 +42,7 @@
     "eslint-plugin-import": "^2.2.0",
     "mocha": "^3.0.2",
     "mocha-eslint": "^3.0.1",
-    "nocms-plugin-directory-resources": "^4.0.1",
+    "nocms-plugin-directory-resources": "file:../nocms-plugin-directory-resources",
```

Comment:
This should probably be updated before merging.

Reply:
I just left this in so you can run the tests and check if everything works on your machine as well :D